frontend/www/js/omegaup/components/common/Navbar.vue

```diff
@@ -289,10 +331,16 @@ export default class Navbar extends Vue {
   get formattedLoginURL(): string {
     return `/login/?redirect=${encodeURIComponent(window.location.pathname)}`;
   }
+
+  get identitiesNotLogged(): types.AssociatedIdentity[] {
```

Comment:
```suggestion
  get identitiesNotLoggedIn(): types.AssociatedIdentity[] {
```

["log"](https://www.merriam-webster.com/dictionary/log#dictionary-entry-2) y ["log in"](https://www.merriam-webster.com/dictionary/log%20on) son cosas distintas :S

Reply:
Listo